inlang/source-code/paraglide/paraglide-js/src/cli/commands/init.test.ts

```diff
@@ -810,21 +853,20 @@ const mockUserInput = (testUserInput: any[]) => {
 const mockFiles = (files: memfs.NestedDirectoryJSON) => {
 	const _memfs = memfs.createFsFromVolume(memfs.Volume.fromNestedJSON(files))
 	const lixFs = createNodeishMemoryFs()
```

Comment:
not sure we should use another memfs, why dont you test with our own memoryFS?

Reply:
not all methods are available that are required and/or the memory fs was not able to be initiated via a JSON for testing

Reply:
@samuelstroschein  can we document this? if just the json format is missing i can add this to the snapshot as a feature in a few lines.

Reply:
@janfjohannes sure, coordinate with @LorisSigrist who owns this code now

Reply:
@janfjohannes The JSON initialisation would be greatly appreciated. I tried doing it with `fromSnapshot`, but that doesn't seem to work unless `fsStat` is provided, which I don't feel like hand-authoring.

Apart from that, I believe lixFs should cover all uses in Paraglide, so ditching memfs would be feasable

Reply:
@LorisSigrist  ok i created a ticket for the json format we need for testing , will ping you when available, lets merge this as is